### ISSUE TITLE
com.vikktorn.picker:picker:1.0.0 library not working  

### ISSUE THREAD

**Issue by hemant-novotrax** [vikktorn/ExampleCountryPicker#13]
implementation 'com.vikktorn.picker:picker:1.0.0'
 not working because of now jcenter not working
https://jcenter.bintray.com/com/vikktorn/picker/picker/1.0.0/picker-1.0.0.aar
404

Please give me any alternetive if availble

**Comment by rehmanjameel** [comment]
Yes same issue with me.. isn't there any way to make it available for use?
